### ISSUE TITLE
Pivot: JAWS reads “Use JAWS key plus Alt + M to move the control element.” This is confusing.

### ISSUE THREAD

**Issue by gggjtgxing** [OfficeDev/office-ui-fabric-react#776]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Describe the issue:
When focus on one pivot item, JAWS reads 'Use JAWS key plus Alt plus M to move the control element', but using these keys are not working.
#### Expected behavior:
Remove 'Use JAWS key plus Alt plus W to move t control element'.

**Comment by dzearing** [comment]
@cschlechty thoughts?

**Comment by mdahamiwal** [comment]
There are two things in Pivot to fix from accessibility POV:
1. Remove `aria-controls` from Tab items. 
    * It is only supported with JAWS+firefox. For other browsers its just a unnecessary noise. Refer to [this](http://juicystudio.com/article/aria-controls-lack-support.php) for more info.
    * According to [tabpanel](https://www.w3.org/TR/wai-aria-1.1/#tabpanel) design pattern this relationship can be created using `aria-labelledby` on tab panel. We are doing both which is not required.

2. All tabs items have same `id` attribute value instead of unique Ids
   This fails the HTML validation. When tabpanel is focussed the aria-labelledby attribute points to all the tab labels. 

I was about to fix 2. but given that 1. is a simple fix i have included it in changes.

**Comment by gggjtgxing** [comment]
Thank you !

**Comment by cschlechty** [comment]
The key suggestions is a JAWS specific hint from JAWS. We should focus on correctly wiring the control, and file a bug to JAWS if there's specific AT bugs.
